src/datasources/cache/__tests__/fake.cache.service.ts

```diff
@@ -23,6 +23,10 @@ export class FakeCacheService implements ICacheService {
 
   deleteByKey(key: string): Promise<number> {
     delete this.cache[key];
+    this.set(
```

Comment:
We are missing an `await` here.  `set` does mostly sync work but in the end returns an empty promise. However, this function shouldn't be required to know that most of the work is done synchronously.

Reply:
Changed in [11b5c6c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/11b5c6c32aecfed991a309d512ca0a9fce3b3346)

(To be able to use an `await` statement here, `deleteByKey` has to be marked as `async` also)